test/android/core_internal/javatests/src/org/xwalk/core/internal/xwview/test/XWalkViewInternalTestBase.java

```diff
@@ -241,7 +241,30 @@ protected void setUp() throws Exception {
             @Override
             public void run() {
                 mXWalkViewInternal = new XWalkViewInternal(activity, activity);
-                getActivity().addView(mXWalkViewInternal);
+                // Create new FrameLayout as a container to add XWalkViewInternal and
+                // ContentViewRenderView, it is just for test case
+                // "OnShowOnHideCustomViewTest.testOnShowCustomViewAndPlayWithHtmlControl".
+                // The play button should be rendered above the video right in
```

Comment:
Since it is just for special cases, any more explanation about why add the change here?